traffic_ops/testing/api/v3/tc-fixtures.json

```diff
@@ -4822,22 +4822,28 @@
     ],
     "serverCapabilities": [
         {
-            "name": "foo"
+            "name": "foo",
+            "description": "server capability foo"
```

Comment:
This won't fail tests, so I feel like I should call it out so it's not accidentally overlooked - extraneous fields shouldn't be present in the testing data.